--- neo4j-enterprise-3.1.0-BETA1/conf/neo4j.conf	2016-10-06 06:03:07.000000000 +0200
+++ core_1/conf/neo4j.conf	2016-10-06 12:17:19.000000000 +0200
@@ -28,8 +28,8 @@
 # calculated based on available system resources.
 # Uncomment these lines to set specific initial and maximum
 # heap size in MB.
-#dbms.memory.heap.initial_size=512
-#dbms.memory.heap.max_size=512
+dbms.memory.heap.initial_size=128
+dbms.memory.heap.max_size=512
 
 # The amount of memory to use for mapping the store files, in bytes (or
 # kilobytes with the 'k' suffix, megabytes with 'm' and gigabytes with 'g').
@@ -39,13 +39,13 @@
 # rest for the page cache.
 # The default page cache memory assumes the machine is dedicated to running
 # Neo4j, and is heuristically set to 50% of RAM minus the max Java heap size.
-#dbms.memory.pagecache.size=10g
+dbms.memory.pagecache.size=1g
 
 # Enable online backups to be taken from this database.
 #dbms.backup.enabled=true
 
 # To allow remote backups, uncomment this line:
-#dbms.backup.address=0.0.0.0:6362
+dbms.backup.address=0.0.0.0:6371
 
 #*****************************************************************
 # Network connector configuration
@@ -70,15 +70,15 @@
 # Bolt connector
 dbms.connector.bolt.enabled=true
 #dbms.connector.bolt.tls_level=OPTIONAL
-#dbms.connector.bolt.listen_address=:7687
+dbms.connector.bolt.listen_address=0.0.0.0:7691
 
 # HTTP Connector
 dbms.connector.http.enabled=true
-#dbms.connector.http.listen_address=:7474
+dbms.connector.http.listen_address=0.0.0.0:7481
 
 # HTTPS Connector
 dbms.connector.https.enabled=true
-#dbms.connector.https.listen_address=:7473
+dbms.connector.https.listen_address=0.0.0.0:7371
 
 # Number of Neo4j worker threads.
 #dbms.threads.worker_count=
@@ -144,35 +144,35 @@
 # CORE - Core member of the cluster, part of the consensus quorum.
 # EDGE - Edge member of the cluster, an eventually-consistent read-only instance of the database.
 # To operate this Neo4j instance in Core-Edge mode as a core member, uncomment this line:
-#dbms.mode=CORE
+dbms.mode=CORE
 
 # Expected number of Core machines in the cluster.
-#core_edge.expected_core_cluster_size=3
+core_edge.expected_core_cluster_size=3
 
 # A comma-separated list of the address and port for which to reach all other members of the cluster. It must be in the
 # host:port format. For each machine in the cluster, the address will usually be the public ip address of that machine.
 # The port will be the value used in the setting "core_edge.discovery_listen_address".
-#core_edge.initial_discovery_members=localhost:5000,localhost:5001,localhost:5002
+core_edge.initial_discovery_members=localhost:5001,localhost:5002,localhost:5003
 
 # Host and port to bind the cluster member discovery management communication.
 # This is the setting to add to the collection of address in core_edge.initial_core_cluster_members.
 # Use 0.0.0.0 to bind to any network interface on the machine. If you want to only use a specific interface
 # (such as a private ip address on AWS, for example) then use that ip address instead.
 # If you don't know what value to use here, use this machines ip address.
-#core_edge.discovery_listen_address=:5000
+core_edge.discovery_listen_address=:5001
 
 # Network interface and port for the transaction shipping server to listen on. If you want to allow for
 # messages to be read from
 # any network on this machine, us 0.0.0.0. If you want to constrain communication to a specific network address
 # (such as a private ip on AWS, for example) then use that ip address instead.
 # If you don't know what value to use here, use this machines ip address.
-#core_edge.transaction_listen_address=:6000
+core_edge.transaction_listen_address=:6001
 
 # Network interface and port for the RAFT server to listen on. If you want to allow for messages to be read from
 # any network on this machine, us 0.0.0.0. If you want to constrain communication to a specific network address
 # (such as a private ip on AWS, for example) then use that ip address instead.
 # If you don't know what value to use here, use this machines ip address.
-#core_edge.raft_listen_address=:7000
+core_edge.raft_listen_address=:7001
 
 #*****************************************************************
 # Core-Edge Additional Configuration Options
